ui-tests/cypress/support/pageObjects/pages/templates/AddPage.js

```diff
@@ -38,6 +38,9 @@ export default class AddPage extends Content {
 
   getJsonConfigArea() {
     return this.getFormArea()
+               .contains("JSON configuration")
```

Comment:
Please, avoid using _contains_; see previous comment

Reply:
For these `contains` issues, the environment must have changed because many of these identifiers aren't there anymore. A different solution would be to use more `children` and `find` selectors, along with `eq` indexes. Would that be preferable?

Reply:
I would say yes, as it should be more reliable than a text that might change even just changing the page language